app/grandchallenge/components/models.py

```diff
@@ -2592,5 +2605,18 @@ def import_status_context(self):
             return "info"
 
     @property
-    def import_in_progress(self):
-        return self.import_status == ImportStatusChoices.INITIALIZED
+    def animate(self):
```

Comment:
Why change the states when animation is done? Given how the import task works I think the previous was correct.